vulnerabilities/importer.py

```diff
@@ -458,6 +460,22 @@ def clean_summary(self, summary):
         return summary
 
     def to_dict(self):
+        is_adv_v2 = (
+            self.severities
+            or self.references_v2
+            or (self.affected_packages and isinstance(self.affected_packages[0], AffectedPackageV2))
```

Comment:
Also do we need advisory_id in json as well ?

Reply:
Yes we can, but i suggest we do it in separate pr since that may require adjusting tests for our existing v2 pipelines.

Reply:
We also have `advisory_id` now https://github.com/aboutcode-org/vulnerablecode/pull/1971/commits/737d94a75f6c9825a2693ba3a07e961ab6e9380e